docs/quickstart.rst

```diff
@@ -1,114 +1,271 @@
+.. _quickstart-guide:
+
 Quick Start Guide
 =================
 
-This guide goes through the process step-by-step and provides assistance in how
-to bootstrap and configure the system.
+Mirantis Managed Services Tooling is a software platform that
+enables Mirantis Managed Services engineers to access remote cloud
+environments on the customer site and monitor and troubleshoot
+these environment as required.
+
+This document provides step-by-step instructions on how to configure
+the required environment for Mirantis Managed Services Tooling.
+
+.. _qs-prerequisites:
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You can install the Managed Services Tooling from an ISO imaged provided
+by the automated build system. Alternatively, you can build your own ISO
+images. A pre-built image already includes a set of required packages. If you
+decide to build your own ISO, you must add all the required packages
+manually.
+
+.. note::
+   The automated build ISOs are not yet supported. Therefore, you must build
+   the ISO manually.
+
+The following table describes the prerequisites for the manually built ISO.
+
+.. list-table:: **Prerequisites for the manually built ISO**
+   :widths: 10 25
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - Building environment
+     - Ubuntu 16.04 x64 with the following software packages:
+
+       * Packer 0.11.0
+       * QEMU-KVM 2.5
+       * GnuPG 1.4
+   * - Base operating system for the ISO image
+     - CentOS 7 or Ubuntu 16.04 x64.
+
+       Although, you can use Ubuntu as a base operating system for
+       the Mirantis Managed Services Tooling bootstrap image, CentOS is
+       preferred.
+
+       The current supported image is Centos 7 Minimal ISO.
+
+.. seealso::
+
+   * :ref:`qs-required-packages`
+   * `CentOS Minimal ISO mirrors <http://isoredirect.centos.org/centos/7/isos/x86_64/>`
+
+.. _qs-required-packages:
+
+Required packages
+-----------------
 
+If you build the Managed Services Tooling ISO manually, you need to install
+a specific set of packages required for the ISO. The latest packages are used
+automatically.
 
-Build Installation ISO
-----------------------
+The Managed Services Tooling ISO must include the following packages:
+
+.. list-table:: **Required packages**
```

Comment:
Does the **building environment** need an Internet connection or a connection to a repo with the required packages?